library/src/main/kotlin/com/chuckerteam/chucker/internal/data/entity/HttpTransaction.kt

```diff
@@ -53,7 +53,8 @@ internal class HttpTransaction(
     @ColumnInfo(name = "responseHeadersSize") var responseHeadersSize: Long?,
     @ColumnInfo(name = "responseBody") var responseBody: String?,
     @ColumnInfo(name = "isResponseBodyEncoded") var isResponseBodyEncoded: Boolean = false,
-    @ColumnInfo(name = "responseImageData") var responseImageData: ByteArray?
+    @ColumnInfo(name = "responseImageData") var responseImageData: ByteArray?,
+    @ColumnInfo(name = "graphQlOperationName") var graphQlOperationName: String?,
```

Comment:
Just curious, but wouldn't we need to include a check for this parameter in the `fun hasTheSameContent()`?

Reply:
Good call, I've added it 👍